module-2/backend/week_two.md

```diff
@@ -14,7 +17,15 @@ end
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
 
+  Team.all, Team.first, Team.last, Team.find_by(attribute: value), Team.count.  These methods are inherited from ActiveRecord.
+  
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+  
+  name = Team.find_by(id: 4).name
+  owner_id = Team.find_by(id: 4).owner_id
+  Owner.find_by(id: owner_id)
```

Comment:
Looks good! Remember, if you're doing a `find_by(id: 4)`, you can replace that with a `find(4)` with the same results. 👍